admin_guide/http_proxies.adoc

```diff
@@ -0,0 +1,120 @@
+= Working with HTTP Proxies
+{product-author}
+{product-version}
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:description: This should be a clear and concise description of the topic.
+:keywords: comma, separated, keywords, list
+
+toc::[]
+
+== Overview
+Production environments can deny direct access to the web and instead, have an HTTP or
+HTTPS proxy available. Configuring OpenShift builds and deployments to
+use these proxies is as simple as setting standard environment variables in
+configuration or JSON files.
+
+== Importing Image Streams
```

Comment:
@codificat In looking at this as a whole, i think it's coming together. However, the "Importing Image Streams" now is empty as per your suggestion above. Is there an alternate procedure for these that I can throw into the docs? Or would someone else be a better person to ask? Thanks.